tests/test_ppx.expected.ml

```diff
@@ -0,0 +1,447 @@
+let f0 dbh =
```

Comment:
How are these `expect` tests supposed to be updated? I assume you didn't write the expected result by hand.

Reply:
I wrote an empty `tests/test_ppx.expected.ml`, ran `make test`, and then copied `_build/default/test_ppx.result` into `tests/test_ppx.expected.ml`. Obviously after the last step I did a cursory review of the file to make sure the generated code from the current build was sensible.

Reply:
Could you make a `make` target or so to recreate these files, in case we change how they compile so we don't have to dig out your comment in this PR in the future?